### PR TITLE
Refactor(cv_container_v3): Adding image bundle dictionary to the output.

### PR DESCRIPTION
**Pull request by Vibhu-gslab**
## Change Summary

Adding integration tests for cv_conatiner_v3

## Related Issue(s)

Fixes #<ISSUE ID>

## Component(s) name

`arista.cvp.cv_conatiner_v3`

## How to test
molecule converge -s cv_container_v3

### User Checklist

- N/A

### Repository Checklist

- [ ] My code has been rebased from devel before I start
- [ ] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [ ] My change requires a change to the documentation and documentation have been updated accordingly. (check the box if not applicable)
- [ ] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/cvp/molecule/cv_container_v3/converge.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
```
Comment (sugetha24) [inline ansible_collections/arista/cvp/molecule/cv_container_v3/converge.yml]: Can we follow the same format as the other molecule tests?
move these tests into a separate playbook and import it in converge.yml?

File: ansible_collections/arista/cvp/molecule/cv_container_v3/converge.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - arista.cvp
+  vars:
+    CONTAINERS_PROVISION_ADD_IMAGE:
+        Test_container:
+            configlets: []
+            imageBundle: EOS-4.25.4M
+            parentContainerName: Tenant
+
+  tasks:
+    - name: "Build Container topology on {{inventory_hostname}}"
```
Comment (sugetha24) [inline ansible_collections/arista/cvp/molecule/cv_container_v3/converge.yml]: Can we add tests for the following?
1. Create a 3 tier topology
2. Delete the above topology
3. Create a 3 tier topology and attach configlet to one of the containers and remove it
4. Create a 3 tier topology and attach image bundles to one of the containers and remove it
5. Play around with `apply_mode: loose` and `apply_mode: strict`. 
Refer to [cv_container_v3](https://cvp.avd.sh/en/stable/docs/how-to/v3/cv_container_v3/) for more info on how `apply_mode` works.
NOTE: `apply_mode` works for both configlets and imageBundle, so we need to test both

File: ansible_collections/arista/cvp/molecule/cv_container_v3/converge.yml (hunk 1 above)
Comment (Vibhu-gslab) [reply ansible_collections/arista/cvp/molecule/cv_container_v3/converge.yml]: Fixed!

File: ansible_collections/arista/cvp/molecule/cv_container_v3/converge.yml (hunk 2 above)
Comment (Vibhu-gslab) [reply ansible_collections/arista/cvp/molecule/cv_container_v3/converge.yml]: Fixed!

File: ansible_collections/arista/cvp/molecule/cv_container_v3/test_cv_container_v3_configlet.yaml [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+    TOPOLOGY_3_TIER:
+      TEAM01:
+        parentContainerName: Tenant
+      TEAM01_DC:
+        parentContainerName: TEAM01
+      TEAM01_LEAFS:
+        parentContainerName: TEAM01_DC
+        configlets:
+          - ATD-INFRA
```
Comment (sugetha24) [inline ansible_collections/arista/cvp/molecule/cv_container_v3/test_cv_container_v3_configlet.yaml]: can we use our own configlet instead? I don't want to rely on ATD configlets

Comment (Vibhu-gslab) [reply ansible_collections/arista/cvp/molecule/cv_container_v3/test_cv_container_v3_configlet.yaml]: Yes, we can use any configlet, whichever is available on your lab.

File: ansible_collections/arista/cvp/molecule/cv_container_v3/test_cv_container_v3_configlet.yaml [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+        that:
+          - REMOVE_TOPO_3_TIER.container_deleted.changed == true
+          - REMOVE_TOPO_3_TIER.container_deleted.container_deleted_count == 3
+          - REMOVE_TOPO_3_TIER.container_deleted.container_deleted_list == ["TEAM01_LEAFS", "TEAM01_DC", "TEAM01"]
+          - REMOVE_TOPO_3_TIER.container_deleted.success == true
+
+    # TC2 - Create a 3 tier topology and attach configlet to one of the containers and remove it.
+    # Testing:
+    #       1. Adding configlet
+    #       2. Change in configlet with apply_mode: loose
```
Comment (sugetha24) [inline ansible_collections/arista/cvp/molecule/cv_container_v3/test_cv_container_v3_configlet.yaml:87]: can we add a test to change configlet with `apply_mode: strict` as well? This should match the configlets on CVP with what's supplied in `configlet` keyword making adds and removes as necessary.

Comment (Vibhu-gslab) [reply ansible_collections/arista/cvp/molecule/cv_container_v3/test_cv_container_v3_configlet.yaml:87]: Added!

File: ansible_collections/arista/cvp/molecule/cv_container_v3/test_cv_container_v3_configlet.yaml (hunk 3 above)
Comment (Vibhu-gslab) [reply ansible_collections/arista/cvp/molecule/cv_container_v3/test_cv_container_v3_configlet.yaml]: Fixed!

File: ansible_collections/arista/cvp/molecule/cv_container_v3/test_cv_container_v3_configlet.yaml [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+    #       3. Change in configlet with apply_mode: strict
+    #       4. Remove configlet using apply_mode: strict
+    #       5. Removal of container which contains configlet.
+
+    - name: Build Container Topology On {{inventory_hostname}} With Configlet
+      arista.cvp.cv_container_v3:
+        topology: '{{TOPOLOGY_3_TIER}}'
+      register: CONFIGLET_ADD
+
+    - name: Testing Added Configlet
```
Comment (sugetha24) [inline ansible_collections/arista/cvp/molecule/cv_container_v3/test_cv_container_v3_configlet.yaml:103]: Can we get configlet facts from CVP and check if it's applied on the container we want?

Comment (Vibhu-gslab) [reply ansible_collections/arista/cvp/molecule/cv_container_v3/test_cv_container_v3_configlet.yaml:103]: Added!

**Review by sugetha24 — APPROVED** [review]

**Review by noredistribution — APPROVED** [review]
LGTM! worked nicely on AtD!

**Pull request merged**